### PR TITLE
chore: Docusaurus with webpack 5

### PR DESCRIPTION
**Pull request by slorber**
Use an unreleased version of Docusaurus to test the webpack5 support works for this site, detect potential problems, and perf improvements.

See also https://github.com/facebook/docusaurus/pull/4089

### CODE REVIEW

File: web/docusaurus.config.js [hunk 1]
```diff
@@ -13,8 +13,6 @@ module.exports = {
   organizationName: 'supabase', // Usually your GitHub org/user name.
   projectName: 'supabase', // Usually your repo name.
   themeConfig: {
-    forceDarkMode: true,
```
Comment (slorber) [inline web/docusaurus.config.js:16]: these do not exist

**Comment by kiwicopple** [comment]
Great, thanks @slorber - note that you can use this URL to get the preview (we have redirects on the monorepo):

https://docs-git-fork-slorber-master-supabase.vercel.app/docs

There are a few css/formatting issue I can see, probably missing CSS classes in the updated docusaurus theme

**Comment by slorber** [comment]
hi @kiwicopple 

I've opened a similar PR to like 20 Docusaurus sites to validate the Webpack 5 support works decently.
Unfortunately, I won't be able to fix the CSS of those 20 sites 😅 
Once I release the next Docusaurus version with Webpack 5, if you feel this PR can't be merged as is, don't hesitate to commit to my PR (you have write access to my work), complete it or re-open it.

Sometimes upgrading Docusaurus to the latest version breaks some CSS, as we have a quite "implicit API surface" due to any user being able to override the default theme and provide custom CSS. We'll try to improve so that it does not happen too frequently in the future.

**Comment by kiwicopple** [comment]
> I won't be able to fix the CSS of those 20 sites

Understood, thanks for upgrading us! We'll make it look great

**Comment by slorber** [comment]
ready for review, as alpha 75 has been released with webpack 5

I'll let you fix the last CSS details of this PR, or close it and open a new PR if you want, just wanted t let you know the webpack 5 support is ready

**Comment by MildTomato** [comment]
fixing the CSS on this now

**Review by kiwicopple — APPROVED** [review]

**Pull request merged**